src/main/java/core/basesyntax/EmployeeService.java

```diff
@@ -1,10 +1,24 @@
 package core.basesyntax;
 
+import java.util.Comparator;
 import java.util.List;
 import java.util.Set;
+import java.util.TreeSet;
 
 public class EmployeeService {
     public Set<Employee> getEmployByOrder(List<Employee> employees) {
```

Comment:
The method name 'getEmployByOrder' contains a typo. It should be 'getEmployeeByOrder' to correctly reflect that it returns employees ordered by certain criteria.